mephisto/core/hydra_config.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from hydra.core.config_store import ConfigStoreWithProvider
+from mephisto.data_model.blueprint import BlueprintArgs
+from mephisto.data_model.architect import ArchitectArgs
+from mephisto.data_model.crowd_provider import ProviderArgs
+from mephisto.data_model.task_config import TaskConfigArgs
+from dataclasses import dataclass, field
+from omegaconf import MISSING
+from typing import List, Any
+
+config = ConfigStoreWithProvider("mephisto")
+
+
+@dataclass
+class MephistoConfig:
+    blueprint: BlueprintArgs = BlueprintArgs()
+    provider: ProviderArgs = ProviderArgs()
+    architect: ArchitectArgs = ArchitectArgs()
+    task: TaskConfigArgs = TaskConfigArgs()
+
+
+@dataclass
+class ScriptConfig:
```

Comment:
Thoughts on naming this `RunScriptConfig`, almost all python files are scripts, so this adds a bit more distinction? in which case would also have to rename `register_script_config` to `register_run_script_config`